basic/src/test/java/org/jboss/hal/testsuite/test/runtime/elytron/CredentialStoreAliasTestCase.java

```diff
@@ -2,77 +2,145 @@
 
 import org.apache.commons.lang.RandomStringUtils;
 import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.drone.api.annotation.Drone;
 import org.jboss.arquillian.graphene.page.Page;
 import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.hal.testsuite.fragment.config.AddResourceWizard;
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
+import org.jboss.hal.testsuite.creaper.ResourceVerifier;
+import org.jboss.hal.testsuite.dmr.ModelNodeGenerator;
+import org.jboss.hal.testsuite.fragment.runtime.elytron.AliasWizard;
 import org.jboss.hal.testsuite.fragment.shared.util.ResourceManager;
 import org.jboss.hal.testsuite.page.runtime.ElytronRuntimePage;
-import org.jboss.hal.testsuite.test.configuration.elytron.AbstractElytronTestCase;
 import org.jboss.hal.testsuite.test.configuration.elytron.ElytronOperations;
+import org.jboss.hal.testsuite.util.ConfigUtils;
 import org.junit.AfterClass;
 import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.io.IOUtils;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.Constants;
 import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
 import org.wildfly.extras.creaper.core.online.operations.Address;
 import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
 import org.wildfly.extras.creaper.core.online.operations.Values;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+import org.wildfly.extras.creaper.core.online.operations.admin.DomainAdministration;
 
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.concurrent.TimeoutException;
 
+
 @RunWith(Arquillian.class)
 @RunAsClient
-public class CredentialStoreAliasTestCase extends AbstractElytronTestCase {
+public class CredentialStoreAliasTestCase {
 
     private static final String CREDENTIAL_STORE_NAME = "credential_store_" + RandomStringUtils.randomAlphanumeric(7);
+    private static final String PARENT_CREDENTIAL_STORE_NAME = "owning_credential_store_" + RandomStringUtils.randomAlphanumeric(7);
+    private static final String CHILD_CREDENTIAL_STORE_NAME = "depending_credential_store_" + RandomStringUtils.randomAlphanumeric(7);
+
+    private static final String TEST_SERVER = "test-server-" + RandomStringUtils.randomAlphanumeric(7);
+    private static final String TEST_PROFILE = "test_" + RandomStringUtils.randomAlphanumeric(7);
```

Comment:
Can you please change the `TEST_PROFILE` prefix to `test-profile-` for consistency sake.